CHANGELOG.md

```diff
@@ -8,6 +8,9 @@ about writing changes to this log.
 
 ## [Unreleased]
 
+* Replaced slashes from webform labels in GetOrganized filenames.
```

Comment:
“Sanitized GetOrganized filenames” may make more sense for non-programmers who happen to read this.

Reply:
Updated.